block/provider/token_generator.go

```diff
@@ -56,6 +57,7 @@ func (tg *tokenGenerator) readConfig(logger zap.Logger) (err error) {
 
 	path := filepath.Join(config.GetEtcPath(), tg.tokenKID)
 
+	//nolint:gosec
```

Comment:
what is the security error here, is there any other way to fix it?

Reply:
The security error was --> G304: Potential file inclusion via variable.
Reason is if we specify the file path with a variable, there is a risk that an unintended file path will be specified.
The fix is 
To clean up possible bad paths through filepath.Clean() before using.

Reply:
Done